### PR TITLE
Fixing the Windows build/installer and adding documentation for setting up Radius

### PR DESCRIPTION
**Pull request by shawnjk**
Adding documentation on how build the Windows installer, setup the Radius server on Windows, and test the Radius server
Adding pap_challenge_request.pl to Linux build
Fixing the Windows installer so it can be "Run as administrator"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,6 +173,53 @@ Start the RADIUS server
 ### Windows
     net start toopher-freeradius
 
+Testing/Debugging the RADIUS server
+-------------------------
+Uncomment the following lines in `/etc/raddb/clients.conf` (Start -> Toopher -> Toopher-Radius Server -> Edit clients.conf in Windows):
```
Comment (dshafer) [inline README.md]: this should reference `/etc/raddb/toopher_users`, I believe.

File: README.md [hunk 2]
```diff
@@ -151,6 +173,53 @@ Start the RADIUS server
...
+        sudo radiusd -X
+
+* Windows
+
+        net stop toopher-freeradius
+Start -> Toopher -> Toopher-Radius Server -> Run in Debug Mode -> Run as administrator...
+
+Attempt to authenticate the `usr_local` and `toopher_usr_local` users via your VPN. You should be able to see debug output in your terminal from authentication requests to your RADIUS server. 
+
+**Recomment the lines in `/etc/raddb/clients.conf` after you are finished testing.**
```
Comment (dshafer) [inline README.md]: `/etc/raddb/toopher_users`

File: cygwin/build [hunk 3]
```diff
@@ -119,7 +113,7 @@ sed -i 's/= radeapclient/= radeapclient$(EXEEXT)/g' ${FREERAD}/src/modules/rlm_e
 pushd ${FREERAD}
 
 # configure freeradius source
-./configure --disable-shared --enable-static --prefix=${FREERAD_BUILD} \
+LIBS="-llber" ./configure --disable-shared --enable-static --prefix=${FREERAD_BUILD} \
```
Comment (dshafer) [inline cygwin/build:118]: Can you add a comment documenting what problem this fixed?

File: cygwin/build [hunk 4]
```diff
@@ -75,9 +64,14 @@ echo STATICPERL_OPTS is ${STATICPERL_OPTS}
 
 cp -fr ./libperl.a .staticperl/perl/bin/libperl.a
 cp -fr ./libperl.a .staticperl/perl/lib/CORE/libperl.a
-cp -fr ./perl.exe ${PERL}
 
-#clean up 
+
+# add staticperl to the PATH so the script uses the correct perl
+export PATH=${STATICPERL_BUILD}/bin:${PATH}
```
Comment (dshafer) [inline cygwin/build:70]: :facepalm: yeah, this is obviously the better way to do it.

File: README.md (hunk 1 above)
Comment (shawnjk) [inline README.md]: Yep, changed in latest commit.

File: cygwin/build (hunk 3 above)
Comment (shawnjk) [inline cygwin/build:118]: Added a comment in the latest commit.

**Pull request merged**